test/unit/jest.config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+const glob = require( 'glob' ).sync;
+
+// Finds all packages which are transpiled with Babel to force Jest to use their source code.
+const transpiledPackageNames = glob( 'packages/*/src/index.js' )
```

Comment:
This will also pick `docgen` (which shouldn't be transpiled but [at the moment it is](https://github.com/WordPress/gutenberg/pull/13329#discussion_r264232869)).

Reply:
Yes, this is the same story as with transpliling it :) We should address it separately.